completions.go

```diff
@@ -211,7 +212,7 @@ func (c *Command) initCompleteCmd(args []string) {
 				// 2- Even without completions, we need to print the directive
 			}
 
-			noDescriptions := (cmd.CalledAs() == ShellCompNoDescRequestCmd)
+			noDescriptions := cmd.CalledAs() == ShellCompNoDescRequestCmd || GetEnvConfig(cmd, configEnvVarSuffixDescriptions) == configEnvVarDescriptionsOff
```

Comment:
Could you add a test to make sure descriptions are deactivated when either the program variable or the global variable is properly set?

Reply:
1107319c750f917bc3bf1c74a7705fe6e93123be